### PR TITLE
feat: indicative CCT date and modification timestamps

### PR DESCRIPTION
**Pull request by Judge40**
# feat: add create and modify timestamps to CCT calc
Add `created` and `lastModified` timestamps to the CCT calculation
response.

# feat: add cctDate field to CCT calc
Add a `cctDate` field to the CCT calculation detail response.

TIS21-6544

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -213,6 +218,8 @@ components:
         wte:
           type: number
           format: percent
+          minimum: 0
+          maximum: 1
```
Comment (john12321) [inline openapi.yml:222]: I've been using 0-100 range on FE for percentage (so I don't have to manipulate decimal value to display as percent). Is it better for BE to work with 0-1 range?

Comment (Judge40) [reply openapi.yml:222]: It is a more standard approach, though if we're going to end up with 60.0000000007% I'm open to the idea of transforming it on the BE 😆

Comment (Judge40) [reply openapi.yml:222]: (It's also consistent with the trainee profile's WTE field)

Comment (john12321) [reply openapi.yml:222]: And look at the fun we can have with that! 😅
I see the `placement`  `wholeTimeEquivalent` is e.g. `"1.0"` 😎

**Pull request merged**